draft-ietf-quic-tls.md

```diff
@@ -1368,21 +1368,20 @@ In using TLS, the central key schedule of TLS is used.  As a result of the TLS
 handshake messages being integrated into the calculation of secrets, the
 inclusion of the QUIC transport parameters extension ensures that handshake and
 1-RTT keys are not the same as those that might be produced by a server running
```

Comment:
I believe the client_early_traffic_secret hashes the ClientHello, and therefore should be included.

"handshake, 0-RTT, and 1-RTT keys"

Reply:
too late, I read your comment in #2175 and withdraw this comment. Ship it!